CMakeLists.txt

```diff
@@ -40,8 +40,10 @@ set(_itk_build_testing ${BUILD_TESTING})
 set(_itk_build_shared ${BUILD_SHARED_LIBS})
 set(BUILD_SHARED_LIBS OFF)
 set(elastix_GIT_REPOSITORY "https://github.com/SuperElastix/elastix.git")
+set(elastix_GIT_REPOSITORY "https://github.com/thewtex/elastix.git")
 # Tag of commit "COMP: Quick-fix ITKElastix error: 'itkCostFunction.h' file not found"
```

Comment:
@thewtex Can you please keep this line of comment "# Tag of commit "COMP..." in sync with the `elastix_GIT_TAG`? (Or just remove the line of comment, if you think it is too cumbersome to keep it up-to-date.)

Reply:
Yes, removed.